Jenkinsfile

```diff
@@ -490,6 +502,152 @@ az aks create                      \
+(import "manifests/platforms/eks.jsonnet") {
+config:: import "${clusterName}-autogen.json",
+letsencrypt_environment: "staging",
+prometheus+: import "tests/testdata/prometheus-crashloop-alerts.jsonnet",
+}
+"""
+                                        ])
+                                    }
+                                    // DNS setup
+                                    {
+                                        // update glue records in parent zone
+                                        def nameServers = []
+                                        container('aws') {
+                                            def output = sh(returnStdout: true, script: """
+id=\$(aws route53 list-hosted-zones-by-name --dns-name "${dnsZone}" --max-items 1 --query 'HostedZones[0].Id' --output text)
+aws route53 get-hosted-zone --id "\$id" --query DelegationSet.NameServers
+"""
+                                            )
+                                            nameServers = readJSON(text: output)
+                                        }
+                                        insertGlueRecords(dnsPrefix, nameServers, "60", parentZone, parentZoneResourceGroup)
+                                    }
+                                    {
+                                        // Cluster destroy
+                                        // Only if there is a valid Kubernetes configuration
+                                        if (fileExists("${env.WORKSPACE}/.kubeconf")) {
+                                            // Uninstall BKPR
+                                            withEnv([
+                                                "PATH+KUBECFG=${tool 'kubecfg'}",
+                                            ]) {
+                                                // This is required for AWS in order to release/destroy ELB network interfaces.
+                                                sh """
+                                                    #!/bin/bash
+                                                    set +x
+                                                    set -x
+
+                                                    kubecfg -v delete --kubeconfig \$KUBECONFIG kubeprod-manifest.jsonnet
+                                                    kubectl wait --for=delete ns/kubeprod --timeout=10m
+                                                """
+
+                                                // Destroy AWS objects
+                                                container('aws') {
+                                                    withEnv([
+                                                        "PATH+JQ=${tool 'jq'}",
+                                                        "HOME=${env.WORKSPACE}",
+                                                    ]) {
+                                                        sh """
+                                                            #!/bin/bash
```

Comment:
Ditto, I don't believe this contains any bash-specific constructs anymore (an earlier version used `<<<`)

Reply:
Fixed.